python_modules/dagster/dagster/core/definitions/decorators/solid.py

```diff
@@ -53,6 +53,8 @@ def __call__(self, fn):
         if not self.name:
             self.name = fn.__name__
 
+        self._fn = fn
```

Comment:
its not clear how this is useful since the `_Solid` instances are just temporary and this information is not making it in to the `SolidDefinition`